react/package.json

```diff
@@ -59,13 +60,17 @@
     "acorn-object-rest-spread": "^1.1.0",
     "acorn-static-class-property-initializer": "^1.0.0",
     "babel-core": "^6.26.0",
+    "babel-plugin-module-resolver": "^3.1.1",
+    "babel-plugin-transform-import-paths": "^1.0.0",
     "bourbon": "^5.1.0",
```

Comment:
looks good! The only thing I would say is move `bourbon` `bourbon-neat` and `normalize-scss` into build dependencies, so when we compile css in search or EEC we don't need to install them separately